### ISSUE TITLE
SHOUTCAST Serer v1.9.8 Stream error

### ISSUE THREAD

**Issue by nepsalone** [y20k/transistor#241]
unable to stream this link SHOUTCAST Serer v1.9.8 Stream
[http://streaming.hamropatro.com:8399/;]http://streaming.hamropatro.com:8399/;(url)
SHOUTCAST Serer v2.5 working fine
Unable to connect.Device dose not have internet connection
**Stream may be outdated**

**Comment by y20k** [comment]
Hi @nepsalone, 
sorry for the late answer. I was AFK. 

I tried to add 

- `http://streaming.hamropatro.com:8399/;`
- `http://streaming.hamropatro.com:8399/`
- `http://streaming.hamropatro.com:8399`

... all three worked. Are you still experiencing this issue?

**Comment by nepsalone** [comment]
still face same problem in Kitkat devices

**Comment by y20k** [comment]
You are right. I get the same error message, when I try to add this link on Android 4.4. A quick peek in the error logs show the following message: `java.net.ProtocolException: Unexpected status line: ICY 200 OK`.

And googling the error brought me to this post: https://stackoverflow.com/a/22019479

It seems that the streams sends a non-standard header. A possible solution described in the post would involve me switching to a (well known) external library for some of the network operations the app is doing.

I am not sure, if I really want to do this. 

Another solution could be: It is worth a try in my experience to contact the sysadmin for the streaming server and ask him/her to change the header.

**Comment by naitsirch** [comment]
Shoutcast headers cannot be changed and v1.x does not support proxies that might be used as a frontend to response with the correct headers.
